src/main/java/com/ea/restaurant/service/impl/Oauth2ServiceImpl.java

```diff
@@ -73,6 +79,71 @@ public LoginResponseDto loginClient(String clientId, String clientSecret) throws
         .build();
   }
 
+  @Override
+  public void validateToken(String token) throws BadJOSEException, ParseException, JOSEException {
```

Comment:
Add `@Transactional(readOnly = true)` here

Reply:
It is not necessary as this method doesn't access to DB. Also it was moved to the `Oauth2Util`.